install.sh

```diff
@@ -450,7 +450,7 @@ do_install() {
 			esac
 		;;
 
-		centos|rhel|sles)
+		centos|rhel)
```

Comment:
Actually wondering now if we need this option in the switch at all 🤔 (looks like same code is also in the `*)` case 🤔